azure/durable_functions/models/DurableOrchestrationClient.py

```diff
@@ -546,3 +546,59 @@ def _get_raise_event_url(
             request_url += "?" + "&".join(query)
 
         return request_url
+
+    async def rewind(self,
+                     instance_id: str,
+                     reason: str,
+                     task_hub_name: Optional[str] = None,
+                     connection_name: Optional[str] = None):
+        """Return / "rewind" a failed orchestration instance to a prior "healthy" state.
+
+        Parameters
+        ----------
+        instance_id: str
+            The ID of the orchestration instance to rewind.
+        reason: str
+            The reason for rewinding the orchestration instance.
+        task_hub_name: Optional[str]
+            The TaskHub of the orchestration to rewind
+        connection_name: Optional[str]
+            Name of the application setting containing the storage
+            connection string to use.
+
+        Raises
+        ------
+        Exception:
+            In case of a failure, it reports the reason for the exception
+        """
+        id_placeholder = self._orchestration_bindings.management_urls.copy()["id"]
```

Comment:
Hmm...I was hoping the Python SDK would only use the RPC endpoint. The reason we do if/else for JavaScript is for legacy reasons where we had to gradually transition from the old problematic HTTPS mechanism to the faster and more reliable RPC mechanism. From a code maintenance perspective I would prefer we _not_ have two ways of making calls to the extension in the Python SDK, but I guess I haven't been paying close enough attention and I'm only just now realizing we're doing this extra work. Maybe we can do this as a separate PR, but I'd like to get rid of the old logic and just raise exceptions if `rpc_base_url` is not defined.

Reply:
Yeah I too noticed that the else statement was for legacy reasons. I'm happy to remove it now.
@cgillum , just to be sure, your concern is not with the highlighted line (which points to the assignment of id_placeholder) but with the else block in line 587, correct? I can just raise an exception there. Please let me know

Reply:
Yeah, sorry the placement of my comment was a little misleading. I was triggered when I saw this line of code, which is why I added the comment here. :) But yeah, ideally we'd raise an exception there. But again, we don't have to block this PR for this.

Reply:
haha all good, I'd rather not create more tickets to avoid cluttering the kanban board, so I made the change in this PR. If you could give it one last glance to get a final LGTM, that'd be great :) ! @cgillum